recipes/fastqc/meta.yaml

```diff
@@ -6,13 +6,14 @@ about:
 
 build:
   detect_binary_files_with_prefix: true
-  number: 2
+  number: 3
   skip: False
 
 requirements:
   run:
     - openjdk
     - perl
+    - freetype 2.7
```

Comment:
This is strange. fastqc is just a Java program and should not depend on it. Is it missing for openjdk or perl?